src/test/objectstore/store_test.cc

```diff
@@ -3414,6 +3414,212 @@ TEST_P(StoreTest, OmapSimple) {
   }
 }
 
+TEST_P(StoreTest, OmapBench) {
+  // Random dictionary generation
+  static const std::string dict =
+      "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
+  mt19937 gen{random_device{}()};
+  uniform_int_distribution<> dist(0, dict.size() - 1);
+
+  // objects
+  int obj_count = 100000;
+  vector<ghobject_t> objects;
+  objects.reserve(obj_count);
+
+  // Keys
+  int keys_per_object = 100;
+  int key_batch_size = 100;
+  int klen = 64;
+  int vlen = 256;
+
+  int r;
+
+  float min = std::numeric_limits<float>::max();
+  float max = std::numeric_limits<float>::min();
+  float avg = std::numeric_limits<float>::min();
```

Comment:
You are right!  I actually don't understand why total isn't some extremely small number here in testing.  avg makes sense since it's just set eventually set to total / obj_count.  Anyway, I'll fix.